From 58ae5d5c2c7eaa6699f7fe1ad57d9300dc95fb9e Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Tue, 15 May 2012 08:16:30 +0200
Subject: [PATCH] man: overwrite vs. override

'overwrite' conveys the idea of the old text being destroyed.
'override' does not. It is more similar to 'overrule'.
(cherry picked from commit 7cf55dd0700d2b93f18a1bdc50b3c864d0af2246)
---
 man/binfmt.d.xml       |    6 +++---
 man/modules-load.d.xml |    6 +++---
 man/systemctl.xml      |    2 +-
 3 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/man/binfmt.d.xml b/man/binfmt.d.xml
index f5ec805..ca40ad9 100644
--- a/man/binfmt.d.xml
+++ b/man/binfmt.d.xml
@@ -75,14 +75,14 @@
 
                 <para>Each configuration file is named in the style of
                 <filename>&lt;program&gt;.conf</filename>.
-                Files in <filename>/etc/</filename> overwrite
+                Files in <filename>/etc/</filename> override
                 files with the same name in <filename>/usr/lib/</filename>.
-                Files in <filename>/run</filename> overwrite files with
+                Files in <filename>/run</filename> override files with
                 the same name in <filename>/etc/</filename> and
                 <filename>/usr/lib/</filename>. Packages should install their
                 configuration files in <filename>/usr/lib/</filename>, files
                 in <filename>/etc/</filename> are reserved for the local
-                administration, which possibly decides to overwrite the
+                administration, which possibly decides to override the
                 configurations installed from packages. All files are sorted
                 by filename in alphabetical order, regardless in which of the
                 directories they reside, to ensure that a specific
diff --git a/man/modules-load.d.xml b/man/modules-load.d.xml
index e2f7d5c..9b1afc9 100644
--- a/man/modules-load.d.xml
+++ b/man/modules-load.d.xml
@@ -76,14 +76,14 @@
 
                 <para>Each configuration file is named in the style of
                 <filename>&lt;program&gt;.conf</filename>.
-                Files in <filename>/etc/</filename> overwrite
+                Files in <filename>/etc/</filename> override
                 files with the same name in <filename>/usr/lib/</filename>.
-                Files in <filename>/run</filename> overwrite files with
+                Files in <filename>/run</filename> override files with
                 the same name in <filename>/etc/</filename> and
                 <filename>/usr/lib/</filename>. Packages should install their
                 configuration files in <filename>/usr/lib/</filename>, files
                 in <filename>/etc/</filename> are reserved for the local
-                administration, which possibly decides to overwrite the
+                administration, which possibly decides to override the
                 configurations installed from packages. All files are sorted
                 by filename in alphabetical order, regardless in which of the
                 directories they reside, to ensure that a specific
diff --git a/man/systemctl.xml b/man/systemctl.xml
index 72ce8a4..c7eb91c 100644
--- a/man/systemctl.xml
+++ b/man/systemctl.xml
@@ -344,7 +344,7 @@
                                 <term><option>-f</option></term>
 
                                 <listitem><para>When used with
-                                <command>enable</command>, override any
+                                <command>enable</command>, overwrite any
                                 existing conflicting
                                 symlinks.</para></listitem>
 
